fifth_task.py

```diff
@@ -0,0 +1,7 @@
+#5 task
+vip_guests = ['Juli', 'Michael', 'John']
```

Comment:
I mentioned in a task that vip guests can not be changed. But I can do it with this data type